docusaurus/src/components/Callout.module.css

```diff
@@ -0,0 +1,7 @@
+.callout {
+  background: var(--color-dark-blue-30);
+  border-radius: 0.3125rem;
+  width: 100%;
+  margin: 0;
```

Comment:
Is this `margin: 0` necessary? I didn't seem to notice a change when I removed it

Reply:
Nope! I thought it was. Might have been a weird hot reload issue when I was splitting apart css?